### PR TITLE
Switch a bunch of documentation titles to title case

### PR DESCRIPTION
**Pull request by danielbachhuber**

### CODE REVIEW

**Comment by mkaz** [comment]
I honestly don't know what should be the preferred style.

Randomly looking through other documentation I see both, just one example here:
https://make.wordpress.org/core/handbook/best-practices/when-you-become-a-committer/

**Review by gziolo — APPROVED** [review]
I like this patch, let's start moving toward title case to solve the issue 🚀

**Comment by danielbachhuber** [comment]
@gziolo Any ideas on why the build is failing with this?

image

**Comment by danielbachhuber** [comment]
> @gziolo Any ideas on why the build is failing with this?

Nevermind, I've found the issue.

**Comment by chrisvanpatten** [comment]
@danielbachhuber The docs manifest needs to be updated; it contains the doc titles which are extracted from the markdown. `npm run docs:build` locally and it should update `docs/manifest.json` for you.

**Comment by danielbachhuber** [comment]
> @danielbachhuber The docs manifest needs to be updated

Yep, done in 8c5220bd011f294d2488f9a38cdb1f82109084fd

File: docs/designers-developers/developers/packages.md
Comment (chrisvanpatten) [inline docs/designers-developers/developers/packages.md]:
```suggestion
## Using the Packages via npm
```

"npm" should [always be lower-case](https://css-tricks.com/start-sentence-npm/). (`lowercase_npm_dangit()`)

File: docs/designers-developers/developers/tutorials/javascript/scope-your-code.md
Comment (chrisvanpatten) [inline docs/designers-developers/developers/tutorials/javascript/scope-your-code.md]:
```suggestion
# Scope Your Code
```

File: docs/designers-developers/developers/tutorials/javascript/versions-and-building.md
Comment (chrisvanpatten) [inline docs/designers-developers/developers/tutorials/javascript/versions-and-building.md]:
```suggestion
# JavaScript Versions and Build Step
```

**Review by chrisvanpatten — APPROVED** [review]
A few small changes; good to go after that!

File: docs/designers-developers/developers/packages.md
Comment (chrisvanpatten) [reply docs/designers-developers/developers/packages.md]: (As a note typically style guides do often overrule how brands prefer their names capitalized, but given our own `capital_P_dangit()` history I think it's a nice sign of respect to use lower-case here, as they desire!)

File: docs/designers-developers/developers/tutorials/javascript/scope-your-code.md
Comment (danielbachhuber) [reply docs/designers-developers/developers/tutorials/javascript/scope-your-code.md]: This is already fixed.

**Pull request merged**
